packages/peregrine/lib/talons/Image/useResourceImage.js

```diff
@@ -28,7 +28,8 @@ export const useResourceImage = props => {
         return resourceUrl(resource, {
             type,
             height: height,
-            width: width
+            width: width,
+            fit: 'cover'
```

Comment:
This function looks very similar to [this one](https://github.com/magento/pwa-studio/blob/develop/packages/venia-ui/lib/util/images.js#L22-L28) which is used by `generateSrcset` below. We should use the utility if possible otherwise risk future changes getting out of sync.

Reply:
Exported this utility and replaced usage of `resourceUrl` in 39da51e. Good catch 👍